app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -1 +1,25 @@
 /* Fork specific bootstrap overrides go here. This is included before bootstrap. */
+
```

Comment:
doesn't this already exist here? https://github.com/tablexi/nucore-open/blob/uconn-design-updates/app/assets/stylesheets/app/font.scss#L98